port/arduino/adapter/Makefile

```diff
@@ -29,9 +29,9 @@ CORE_HEADERS  =$(addprefix -I$(ROOT_DIR)/, messaging/coap util util/pt include
 CFLAGS += -I. $(CORE_HEADERS) $(LIB_HEADERS) $(DEPS_HEADERS)
 
 ifeq ($(VERBOSE),1)
-	CFLAGS += -DOC_DEBUG -DOC_LOG_L1
+	CFLAGS += -DOC_DEBUG -DOC_LOG_L1 -DOC_LOG_MAXIMUM_LEVEL=8
```

Comment:
I don't think `OC_LOG_L1` exists anymore, I'm 99% sure the arduino port doesn't work.

Reply:
I just added there `-DOC_LOG_MAXIMUM_LEVEL=8` .